src/BlockInfo.h

```diff
@@ -28,9 +28,12 @@ class cBlockInfo
 	/** How much light do the blocks consume? */
 	NIBBLETYPE m_SpreadLightFalloff;
 
-	/** Is a block completely transparent? (light doesn't get decreased(?)) */
+	/** Is a block completely transparent to blocklight? (light doesn't get decreased(?)) */
 	bool m_Transparent;
 
+	/** Does a block disperse sky light? (only relevant for transparent blocks) */
+	bool m_SkylightDispersant;
```

Comment:
Rename to `m_IsSkylightDispersant` or `m_DoesDisperseSkylight` to indicate the bool-ness

Reply:
Should `m_Transparent` not also be `m_IsTransparent` then?

Reply:
Would have been better, but it's been exported to the API for too long so changing it might be dangerous :(

Reply:
The change is incoming.